docs/changelog/118535.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 118535
+summary: "[9.0] Clean up leftover reference of `include_type_name`"
+area: Indices APIs
+type: upgrade
```

Comment:
Just a nit... The docs say that the `>upgrade` label "means an upgrade to an Elasticseach dependency e.g. Java, Lucene, or some other library". Which doesn't apply in this case. Since this change has no user-visible impact (as far as I can see), wouldn't it be more like a `>refactoring` or a `>non-issue` — and no changelog should be required?

Reply:
Thank you!